### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by drobotk**
<3

### CODE REVIEW

**Comment by yuI4140** [comment]
xd

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ TODO: Use rpath to eliminate the need for LD_LIBRARY_PATH
 $ export HOTRELOAD=1
 $ export LD_LIBRARY_PATH="./build/:$LD_LIBRARY_PATH"
 $ ./build.sh
-$ ./build/musualizer <song.ogg>
+$ ./build/musializer <song.ogg>
```
Comment (rexim) [inline README.md:30]: Btw, could you please also remove the `<song.ogg>` while are at it? Musializer does not accept any arguments anymore. I forgot to update it. Thanks!

**Comment by ziebam** [comment]
You could also replace <kbd>/</kbd> with <kbd>.</kbd> on line 10 – no point in having a separate PR for that.

**Comment by rexim** [comment]
Looks goode to me! Thank you for the contribution! :+1:

**Pull request merged**